### PR TITLE
Adds guidance docs to upgrade from old metrics

### PR DESCRIPTION
**Pull request by jmckulk**
**Checklist:**

 
 - [ ] Have you added an explanation of what your changes do and why you'd like them to be included?
 - [ ] Have you updated or added documentation for the change, as applicable?
 - [ ] Have you tested your changes on all related environments with successful results, as applicable?

**Type of Changes:**

 
 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [ ] New feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to change)

**What is the current behavior? (link to any open issues here)**

**What is the new behavior (if this is a feature change)?**

**Other information**:

[ch8955]

### CODE REVIEW

File: docs/content/Upgrade/metrics.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+container-based PostgreSQL deployments. This updated configuration will show
+container specific resource information from your metrics-enabled PostgreSQL
+clusters. By default the metrics stack will include:
+
+- New Grafana dashboards tailored for container-based PostgreSQL deployments
+- Container specific operating system metrics
+- General PostgreSQL alerting rules.
+
+## Updated Monitoring Installer
+The installer for the PostgreSQL Operating Monitoring has been split out into
```
Comment (andrewlecuyer) [inline docs/content/Upgrade/metrics.md:47]: Recommend saying "the PostgreSQL Operating Monitoring infrastructure" here (i.e. add "infrastructure" to the end of "the PostgreSQL Operating Monitoring", which doesn't really sound right by itself).

File: docs/content/Upgrade/metrics.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+a separate set of installers. With each installer (Ansible, a Kubectl job,
+or Helm) you will be able to apply custom configurations through Kubernetes
+ConfigMaps.
```
Comment (andrewlecuyer) [inline docs/content/Upgrade/metrics.md:50]: Thinking we can specify a quick list of things that can be customized, e.g.:

> With each installer (Ansible, a Kubectl job, or Helm) you will be able to apply custom configurations through Kubernetes ConfigMaps.  This includes:
> 
> - Custom Grafana dashboards
> - Custom Prometheus scrape configuration
> - Custom alerting rules

Comment (jmckulk) [reply docs/content/Upgrade/metrics.md:50]: what do you think of this @andrewlecuyer?

- Custom Grafana dashboards and datasources
- Custom Prometheus scrape configuration
- Custom Prometheus alerting rules
- Custom Alertmanager notification configuration

Comment (andrewlecuyer) [reply docs/content/Upgrade/metrics.md:50]: Sounds good to me!

**Review by andrewlecuyer — APPROVED** [review]

**Review by cbandy — APPROVED** [review]
+1 on "stack" vs "infrastructure"

**Pull request merged**